Library/ViewModels/RewardCardViewModel.swift

```diff
@@ -132,7 +132,9 @@ public final class RewardCardViewModel: RewardCardViewModelType, RewardCardViewM
       .map { reward, project, shippingRule in
         guard let locationId = shippingRule?.location.id else { return nil }
 
-        return estimatedShippingText(for: [reward], project: project, locationId: locationId)
+        return project.stats.needsConversion
+          ? estimatedShippingConversionText(for: [reward], project: project, locationId: locationId)
+          : estimatedShippingText(for: [reward], project: project, locationId: locationId)
```

Comment:
Looks like `estimatedShippingConversionText` does its own check for `project.stats.needsConversion`, does it make sense to move this call to `estimatedShippingText(…)` into that function? (It might not)